RFC-010-core-strategy/core-strategy.md

```diff
@@ -0,0 +1,67 @@
+# Executive Summary
+
+The Good Docs Project has grown organically since its creation and has blossomed into a thriving community. Now it’s important to articulate where we are, how we can best support our users and community members, and where we’d like to grow in the future. 
+
+
+## Core Strategy Statement  
+
+_The Good Docs Project educates and empowers people to create high quality documentation by providing them with resources, best practices and tools to enhance their documentation in open source and beyond._
+
+
+## Who we are
+
+We are a community of technical folk. Writers, developers, editors and more. We are passionate about technical writing, documentation and open source.
+
+
+### Our Audience
+
+Our primary audience includes:
+
+* Users of our products
+* Members of our community
+* People seeking to learn more about the technical writing domain
+
+
+### Our friends
+
+We are friends with (and many of us are active in) the [Write The Docs community](https://www.writethedocs.org/). 
+We are not competing with that community. We acknowledge that our interests overlap and that sometimes our activities align. 
+The relationship between our projects is mutually supportive and friendly.
+
+We welcome collaboration and sponsorship opportunities. We like to participate in writing programs and open source programs (like the Google Season of Docs).
+
+
+## What we do
+
+First and foremost, we create documentation templates with supporting meta-documents that help people use and understand the templates. As part of this work, we engage in many other initiatives around docs advocacy, docs education, and docs tooling. 
+
+We produce content to address the goals of engaging in our community, and helping run the project.
+
+We prioritize collaboration in all aspects of our project. We embrace paired-writing, co-working sessions and a consultative approach in all
+our community processes.
+
+We encourage and support each other in learning and gaining experience in the realm of technical writing. 
+
+
+### We don’t compete, we cooperate
+
+We don't compete with other groups and we aren't a business. We are one open-source community looking to make open source, as a whole, better for everyone. 
```

Comment:
I'd go a step further, to make the focus about the participants rather than the project: "We're one open-source community that strives to empower other open source members to make quality documentation for their products."

We're thought leaders (including the nature of our product of templates), not package producers more concerned with direct technical interfacing

Reply:
Thanks for your comments. I've made a change and tried to capture both of your thoughts.